packages/amplify_api_plugin_interface/lib/src/RestAPI/rest_exception.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:amplify_api_plugin_interface/amplify_api_plugin_interface.dart';
```

Comment:
I think long term we are moving to the Dart naming convention of snake case filenames. I can change it for consistency's sake, but it will probably be changed back eventually.

Reply:
Good to know. IMO, better to be consistent and change them at once, but it really does not matter much IMO. Your call.